packages/strapi-plugin-upload/admin/src/components/SortPicker/index.js

```diff
@@ -10,12 +10,12 @@ import Picker from '../Picker';
 
 const SortPicker = ({ onChange, value }) => {
   const orders = {
-    created_at_asc: 'created_at:ASC',
     created_at_desc: 'created_at:DESC',
+    created_at_asc: 'created_at:ASC',
```

Comment:
I don't understand these changes I prefer when it was sorted. Is there any reason for modifying the order?

Reply:
The UI